src/components/QuestionComponent/index.js

```diff
@@ -39,10 +39,21 @@ export default function QuestionComponent({question}) {
 			}
 			return question;
 		});
-		addNewAnswer(newQuestions);
+		setQuestions(newQuestions);
 		event.target.reset();
 	};
 
+	const closeQuestion = id => {
+		const newQuestions = questions.map(question => {
+			if (id === question.id) {
+				question.answered = !question.answered;
+			}
+			return question;
+		});
+		console.log(newQuestions);
```

Comment:
Please delete this console log. In the final PR there should be no console log anymore.